tests/test_files/markdown/lists.md

```diff
@@ -0,0 +1,40 @@
+This is an unordered list with minuses.
+- item 1
+- item 2
+
+
+This is an unordered list with pluses.
++ item 1
++ item 2
+
+
+This is an unordered list with asterisks.
+* item 1
+* item 2
+
+
+This is an unordered list with bullets.
+• item 1
+• item 2
+
+
+This is an ordered list.
+1. item 1
+2. item 2
+6. item 3
```

Comment:
`3. item 3 ` ?

Reply:
That's intentional.  The digits in the markdown do not need to be sequential.